attachment_delete_restrict/tests/test_attachment_delete_restrict.py

```diff
@@ -0,0 +1,238 @@
+# Copyright 2021 Quartile Limited
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo.exceptions import ValidationError
+from odoo.tests import TransactionCase, tagged
+
+
+@tagged("post_install", "-at_install")
+class TestAttachmentDeleteAbstract(TransactionCase):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.param = cls.env["ir.config_parameter"]
+        cls.param.set_param(
+            "attachment_delete_restrict.global_restrict_delete_attachment", "none"
+        )
+        cls.partner_model = cls.env["ir.model"].search([("model", "=", "res.partner")])
+        cls.partner_1 = cls.env["res.partner"].create({"name": "partner_1"})
+        cls.group = cls.env.ref("base.group_user")
+        cls.user_owner = cls.env["res.users"].create(
+            {
+                "name": "test owner user",
+                "login": "test-owner@example.com",
+                "groups_id": [
+                    (
+                        6,
+                        0,
+                        (
+                            cls.env.ref("base.group_user").id,
+                            cls.env.ref("base.group_partner_manager").id,
+                        ),
+                    )
```

Comment:
Let's switch to this style.
```suggestion
                    Command.set(
                        cls.env.ref("base.group_user").id,
                        cls.env.ref("base.group_partner_manager").id,
                    )
```